sdk/core/core-http/src/serviceClient.ts

```diff
@@ -348,20 +349,20 @@ export class ServiceClient {
             queryParameter,
             operationSpec.serializer
           );
-          if (queryParameterValue != undefined) {
+          if (queryParameterValue !== undefined) {
```

Comment:
This is similar to the concern in the core-amqp PR.
Changing `!= undefined` to `!== undefined` loses coverage for null.
Either we don't make this change in this PR and have the library owners follow up or replace `x != undefined` with `x !== undefined && x !== null`

Reply:
Same in other places where the same change is made

Reply:
Done!